mender-convert-modify

```diff
@@ -167,10 +167,11 @@ fi
 # the Debian package is installed later, this folder has to preexist. It is part
 # of the rootfs setup, independently of the software.
 log_info "Creating state folder in the data partition for Mender add-ons"
+run_and_log_cmd "sudo mkdir -p work/rootfs/var/lib"
 run_and_log_cmd "sudo mkdir -p work/rootfs/data/mender-configure"
-run_and_log_cmd "sudo ln -sf /data/mender-configure work/rootfs/var/lib/mender-configure"
+run_and_log_cmd "sudo ln -sf /data/mender-configure work/rootfs/var/lib"
```

Comment:
We are absolutely sure because we are creating `/var/lib` directory few lines above, and because the tests are verifying that the links exist in the resulting images.

I personally like more the syntax with `.../var/lib`.